src/components/front-page-alert/FrontPageAlert.tsx

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { Alert } from '@amsterdam/asc-ui'
+
+import configuration from 'shared/services/configuration/configuration'
+
+import Markdown from '../Markdown'
+
+export const FrontPageAlert = () => {
+  const alertText = configuration.frontPageAlert.text
+
+  return (
+    <>
+      {alertText && (
+        <Alert level="error" outline style={{ marginTop: 5, marginBottom: 30 }}>
```

Comment:
Why are you using inline styles and not like we style all other components?

Reply:
Since it is only the margin that needed to be set. I figured this was shorter. Oscar does this sometimes too.